packages/tools/generator-component/generators/app/templates/__package__.json

```diff
@@ -7,7 +7,7 @@
   "maxBundleSize": "<%= bundlewatchMaxSize %>kB"<% } %>,
   "files": [
     "dist"<% if(config.needsComponentTests) { %>,
-    "test-utils"<% } %> <% if(config.needsTestingApiMocks) { %>,
```

Comment:
Removed a space in the middle, which comes out of the generator as trailing whitespace